nectar/src/database.rs

```diff
@@ -2,18 +2,23 @@ use self::{
     hbit::{HbitFunded, HbitRedeemed, HbitRefunded},
     herc20::{Herc20Deployed, Herc20Funded, Herc20Redeemed, Herc20Refunded},
 };
+#[cfg(test)]
+use crate::StaticStub;
 use crate::{network, network::ActivePeer, swap, swap::SwapKind, SwapId};
 use anyhow::{anyhow, Context, Result};
+use conquer_once::Lazy;
 use serde::{Deserialize, Serialize};
-
-#[cfg(test)]
-use crate::StaticStub;
 use std::{collections::HashSet, iter::FromIterator};
 use time::OffsetDateTime;
 
 mod hbit;
 mod herc20;
 
+static ACTIVE_PEER_KEY: Lazy<Vec<u8>> =
+    Lazy::new(|| serialize(&"active_peer").expect("this slice can be serialized"));
```

Comment:
Alternatively, we could just make these `const` byte slices:

```suggestion
const ACTIVE_PEER_KEY: &[u8] = b"active_peer";
```

Reply:
This would break backward compatibility which was not the intent at the time of this PR.